nkms/keystore/keypairs.py

```diff
@@ -109,3 +110,8 @@ def sign(self, message: bytes) -> bytes:
         """
         signature_der_bytes = API.ecdsa_sign(message, self._privkey)
         return Signature.from_bytes(signature_der_bytes, der_encoded=True)
+
+    def generate_self_signed_cert(self, common_name):
+        # TODO: Let's have a shortcut method for getting the cryptography key(s).
+        cryptography_key = self._privkey.bn_key.to_cryptography_priv_key()
```

Comment:
Oh yes, definitely, I remember thinking about this at some moment, but it seems I forgot